### ISSUE TITLE
Climate_ip 3.2 errors

### ISSUE THREAD

**Issue by KameDomotics** [atxbyea/samsungrac#10]
Hi @atxbyea ,
today I have got this errors in HomeAssistant Log:
(Climate_ip 3.2)

```suggestion

Logger: homeassistant.helpers.entity
Source: /usr/local/lib/python3.8/asyncio/events.py:81 
First occurred: 4:23:22 (8 occurrences) 
Last logged: 6:57:08
Update of climate.samsung_ac is taking over 10 seconds

Logger: custom_components.climate_ip.climate
Source: custom_components/climate_ip/connection_request.py:113 
Integration: climate_ip (documentation, issues) 
First occurred: 7:03:19 (89 occurrences) 
Last logged: 9:28:50
Request execution failed. Stack trace:

Logger: custom_components.climate_ip.climate
Source: custom_components/climate_ip/connection_request.py:135 
Integration: climate_ip (documentation, issues) 
First occurred: 7:03:48 (18 occurrences) 
Last logged: 9:29:02
Execution failed, unknown error

```

**Comment by KameDomotics** [comment]
Today, same error.
The air conditioner has not been switched on or used, it has always remained off (I wasn't even at home)

```
Logger: custom_components.climate_ip.climate
Source: custom_components/climate_ip/connection_request.py:113
Integration: climate_ip (documentation, issues) 
First occurred: 0:21:59 (351 occurrences) 
Last logged: 10:17:18

Request execution failed. Stack trace:

```

```

Logger: custom_components.climate_ip.climate
Source: custom_components/climate_ip/connection_request.py:135
Integration: climate_ip (documentation, issues) 
First occurred: 0:22:28 (64 occurrences) 
Last logged: 10:17:32

Execution failed, unknown error

```

```

Logger: homeassistant.helpers.entity
Source: /usr/local/lib/python3.8/asyncio/events.py:81 
First occurred: 0:25:48 (22 occurrences) 
Last logged: 10:10:52

Update of climate.samsung_ac is taking over 10 seconds

```

**Comment by atxbyea** [comment]
Yeah, I have none of these issues as my AC is running 24\7, hopefully someone else might know a workaround for it.

**Comment by atxbyea** [comment]
are you still having issues @KameDomotics ?

**Comment by KameDomotics** [comment]
Hi, I have always this errors continuously.

In the course of the day these errors are repeated at least a hundred times, in the last few days the problem has worsened considerably.

is there a way to "exclude" this control?

Thanks 

```

Logger: custom_components.climate_ip.climate
Source: custom_components/climate_ip/connection_request.py:113
Integration: climate_ip (documentation, issues) 
First occurred: 19:44:11 (2 occurrences) 
Last logged: 19:44:26

Request execution failed. Stack trace:

```


```

Logger: custom_components.climate_ip.climate
Source: custom_components/climate_ip/connection_request.py:135
Integration: climate_ip (documentation, issues) 
First occurred: 19:44:39 (1 occurrences) 
Last logged: 19:44:39

Execution failed, unknown error

```

**Comment by KameDomotics** [comment]
Hi @atxbyea

I did these tests:

 - created binary_sensor ping on the IP of the air conditioner
 - with the air conditioner OFF, it sometimes disconnects from the wifi network for 30 seconds and then reconnects
 - during this disconnection, it generates the error I described in the past few days
 - with the air conditioning ON, no disconnection and no errors

 
I also checked these things:

 - the Fritzbox 7530 router works fine, I also tried to reset
 - the power supply of the air conditioner is constant and without problems

**Comment by atxbyea** [comment]
I will try turning off my AC for a period of time next time I'm working from home and see if I see the same you are seeing @KameDomotics

**Comment by atxbyea** [comment]
I believe this is now fixed in 3.5.1
